websites/N/Night.design/presence.ts

```diff
@@ -0,0 +1,116 @@
+const presence = new Presence({
+	clientId: "1031730262092042321",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/4mDvKNa.png",
+	};
+
+	presenceData.startTimestamp = Math.floor(Date.now() / 1000);
+
+	const pathname = document.location.pathname.slice(3);
+
+	switch (pathname) {
+		case "/home":
+			presenceData.details = "View the Homepage";
+			break;
+		case "/resources":
+			presenceData.details = "Search...";
+			break;
+		case "/settings/account":
+			presenceData.details = "Change my Account Settings";
```

Comment:
The wording of the details is a bit strange.
For example, I think a better way to word this one would be "Changing Account Settings" (basically, using '-ing' verbs, and removing 'my')